src/psyclone/tests/psyir/backend/fortran_test.py

```diff
@@ -565,14 +567,14 @@ def test_fw_binaryoperator(fort_writer):
         "subroutine tmp(a,n)\n"
         "  integer, intent(in) :: n\n"
         "  real, intent(out) :: a(n)\n"
-        "    a = sign(1.0,1.0)\n"
+        "    a = {0}(1.0,1.0)\n"
```

Comment:
This is not valid Fortran in the case of the SUM intrinsic:

    result = SUM (array [,dim] [,mask])

Reply:
Created separate test.